src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -172,6 +173,10 @@ public function addSkippedTest($test, \Exception $e, $time)
     public function startTest($test)
     {
         if (-2 < $this->state && ($test instanceof \PHPUnit_Framework_TestCase || $test instanceof TestCase)) {
+            if (null !== $test->getTestResultObject()) {
```

Comment:
`getTestResultObject()` can return `null`, for example, when the test case leads to a PHP warning. In this case, `$test` is an instance of `PHPUnit\Framework\WarningTestCase` which does not emit a test result object.